tests/mytests.spec.js

```diff
@@ -0,0 +1,51 @@
+const { test, expect } = require('@playwright/test');
+const exp = require('constants');
+
+async function login (page) {
```

Comment:
Well done!
There is a mechanism so called hooks - which can be used to do necessary preconditions before each test - [beforeEach](https://playwright.dev/docs/api/class-test#test-before-each-1). check out - there some others as well